draft-ietf-quic-transport.md

```diff
@@ -1749,6 +1749,13 @@ values for 0-RTT.  This includes initial_max_data and either
 initial_max_streams_bidi and initial_max_stream_data_bidi_remote, or
 initial_max_streams_uni and initial_max_stream_data_uni.
 
+It is OPTIONAL for a server to be able to recover the previously sent values of
+the max_idle_timout, max_udp_payload_size, and disable_active_migration
+parameters. Lowering the values of these parameters while also accepting 0-RTT
```

Comment:
I know that OPTIONAL and RECOMMENDED are in RFC 2119, but they encourage the passive voice, .

```suggestion
A server MAY choose to recover the previously sent values of the
max_idle_timout, max_udp_payload_size, and disable_active_migration parameters
and reject 0-RTT if it selects smaller values.
Lowering the values of these parameters while also accepting 0-RTT
```

Reply:
I agree that active voice is better than passive voice. However, I think this suggested change changes the meaning slightly: saying the "server MAY choose to recover" refers to the server making a choice at the time it would be recovering, while "OPTIONAL for a server to be able to recover" refers to the server making a choice at the time it is storing (or otherwise encoding) parameters to be able to recover at a future point in time. I'm trying to be more clear that the option (whether active voice MAY or passive voice OPTIONAL) is when storing the transport parameters, not when recovering.

Reply:
Change "MAY choose to recover" to "MAY store and recover". @nharper, any other way of making this active is also good.

Reply:
"MAY choose to store and recover" sounds good to me.